### DISCUSSION TITLE
Using USB connection

### DISCUSSION THREAD

**Discussion by laenzlinger** [piramja/inventree-niimbot-plugin#3] [Q&A]
Is it possible to use USB instead of bluetooth to connect the Niimbot printer.

I was able to print to usb with https://github.com/AndBondStyle/niimprint 

 I am running Inventree in a docker container. Is it even possible to access USB or bluetooth from within the docker container?

**Comment by piramja** [comment]
Both Bluetooth and USB should generally work with this plugin.

But you can't access the host USB from within a docker container out of the box. There might be some workaround tho 🤷🏽‍♂
